packages/dds/tree/src/util/testChangeRebaser.ts

```diff
@@ -0,0 +1,130 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ChangeRebaser } from "../rebase";
+
+interface outputType {
+    "diffRebaseOrder": string | any[];
+    "diffComposeOrder": string | any[];
+    "nestedComposeRebaseOrder": string | any[];
+    "doUndoPair": string | any[];
+    "sandwichRebase": string | any[];
+    "changeWithInverse": string | any[];
+}
+export function testChangeRebaser<TChange>(rebaser: ChangeRebaser<TChange>,
+    changes: ReadonlySet<TChange>,
+    isEquivalent: (a: TChange, b: TChange) => boolean): outputType {
+    const rebase = rebaser.rebase.bind(rebaser);
+    const compose = rebaser.compose.bind(rebaser);
+    const invert = rebaser.invert.bind(rebaser);
+
+    const output: outputType = {
+        diffRebaseOrder: "PASSED",
+        diffComposeOrder: "PASSED",
+        nestedComposeRebaseOrder: "PASSED",
+        doUndoPair: "PASSED",
+        sandwichRebase: "PASSED",
+        changeWithInverse: "PASSED",
+    };
```

Comment:
I think we'll also want to add checks for:
- compose([ε, A]) => A && compose([A, ε]) => A
- rebase(A, ε) => A
- rebase(ε, A) => ε
- invert(ε) => ε

Where ε = compose([])